Source/missiles.h

```diff
@@ -119,7 +119,7 @@ struct Missile {
 	bool _miLightFlag;
 	bool _miPreFlag;
 	uint32_t _miUniqTrans;
-	int _mirange; // Time to live for the missile in game ticks, oncs 0 the missile will be marked for deletion via _miDelFlag
+	int _miduration; // Time to live for the missile in game ticks; once 0, the missile will be marked for deletion via _miDelFlag

```

Comment:
```suggestion
	/** @brief Time to live for the missile in game ticks; once 0, the missile will be marked for deletion via _miDelFlag */
	int duration;
```

Do we still need the `_mi` syntax? For `position` and `limitReached` we also didn't use it.